roles/test_operator/tasks/run-test-operator-job.yml

```diff
@@ -57,6 +57,20 @@
       {{ testjob.attempts == (cifmw_test_operator_timeout / 10) | round | int }}
   when: not cifmw_test_operator_dry_run | bool
 
+- name: Get inormation about PVC that stores the logs
```

Comment:
you can push that in a block in order to group the two actions under the same `when` conditions. That would be better imho :)

Reply:
also inormation -> information

Reply:
Ack, thank you @cjeanner and @lewisdenny. Fixed.